docs/docs/guides/write-tasks.md

```diff
@@ -0,0 +1,114 @@
+# Write Tasks
+
+Director is a wrapper around Celery, so creating tasks with it is *almost* the same
+as creating tasks for pure Celery.
+
+## Create a task
+
+In pure Celery you had to create a Celery application object (`app = Celery(...)`) and use
+the `app.task()` decorator to transform Python function into Celery tasks.
+
+This work has already be done for you in Director, so you just have to transform your function
+using the `director.task` decorator :
+
+```python
+# tasks/example.py
+from director import task
+
+
+@task(name="TASK_EXAMPLE")
+def my_task(*args, **kwargs):
+    pass
+```
+
+!!! warning Task naming
+    The `name` parameter in the task decorator is mandatory. Because it will be used in the YAML
+    file to [combine tasks into workflows](build-workflows.md), this name **must be unique**.
+
+## Task signature
+
+To simplify the tasks creation, and to allow multiple workflows to reuse the same task, their
+signature is always the same : `(*args, **kwargs)`.
+
+The **kwargs** dictionnary can be used to [handle the payload](use-payload.md#handle-payload)
+while **args** contains the results of the task parents (of course args is empty if your task
+is at the beginning of a workflow).
+
+!!! info "Technical explanation"
+    In Celery the developer can decide if a task is able to received the result of its parents or
```

Comment:
to received -> to receive
also, I suggest:
_to receive or not the result of its parents with 2 methods_